aioredisqueue/queue.py

```diff
@@ -160,12 +170,18 @@ def _ack(self, task_id, method='multi'):
             args=[task_id],
         )
 
-    async def _requeue(self, before=None):
+    async def _requeue(self, now, before=-1):
```

Comment:
I think that should become a public member with no required arguments. So if the program have crashed, I can just run an instance of Queue without auto requeueing and manually call `_requeue()`.